src/dune_rules/dune

```diff
@@ -46,3 +46,15 @@
  (mode fallback)
  (action
   (copy# setup.defaults.ml setup.ml)))
+
+(rule
+ (mode promote)
+ (targets setup.defaults.ml)
```

Comment:
why do we need to generate the fallback now? Shouldn't it just have `None` for every default value.